### ISSUE TITLE
Final edit of BODS docs for v0.4 release

### ISSUE THREAD

**Issue by kd-ods** [openownership/data-standard#479]
The [proposed implementations](https://github.com/openownership/data-standard/discussions) for the 0.4 features mean:

 - structural changes in the schema
 - updating the conceptual model
 - terminology changes
 
This means that every page of the docs will need a thorough edit to bring language and concepts into line.

**Comment by kd-ods** [comment]
I've been considering the extent of the docs changes for BODS 0.4. The wholesale edit should maybe go hand-in-hand with a revision to the documentation's structure. 

In particular, I think we don't do a good job at the moment of communicating that the data standard has requirements that go beyond those represented by the schema. That is: the data standard consists of a data schema plus additional requirements (e.g. that entities and persons that have a relationship between them should appear before the relationship statement in a dataset).

In order that the extensive editing work can be shared out, i think we therefore need to do two prior pieces of work:

- [ ] Review existing documentation and structure and produce a documentation workplan
- [ ] Improve the [style guide](https://github.com/openownership/bods-dev-handbook/blob/main/style_guide.md)

**Comment by StephenAbbott** [comment]
Here are the tickets for updating the homepage and primer section pages https://github.com/openownership/data-standard/issues/416 and updating images in primer section https://github.com/openownership/data-standard/issues/324

**Comment by kd-ods** [comment]
I'm making our work-in-progress [documentation workplan visible here](https://docs.google.com/spreadsheets/d/1gaARj0p7UShhDcHP83RFoRXPSstM8Zc4al5UZRA6AJQ/edit?usp=sharing).

**Comment by kd-ods** [comment]
We have enabled a build of the documentation on the main branch here: 

https://standard.openownership.org/en/main/

**Comment by kd-ods** [comment]
@kathryn-ods has done a lot of work to track and manage the documentation updates. Dedicated tickets tagged with 'documentation content' are now open and on the project tracker. 

For that reason, I'm going to rename this ticket to 'Final edit of BODS docs...' and treat it as a parent ticket (putting it back into the Ready column and punting it down the priority order).

**Comment by kathryn-ods** [comment]
Currently the docs for 0.0.1, 0.0.2, 0.0.3 are all 404 ing - should these be removed from the docs?

**Comment by kd-ods** [comment]
They only(!) 404 when an equivalent page doesn't exist in that version of the docs.

So - for example - if you visit this 0.0.1 page: https://standard.openownership.org/en/0.0.1/provenance.html

... then switch to 0.0.2, then it's fine.

Ideally, we should set up the 404 redirects to display the index page for the relevant version of the docs. I think that's what's happening when you you go from the 0.0.1 page above to the equivalent 0.3 page.

**Comment by kathryn-ods** [comment]
Thanks @kd-ods I've set up a page redirect https://docs.readthedocs.io/en/stable/user-defined-redirects.html#id2 for all broken pages to redirect back to the index page for that version. 

In theory this should work across all versions and languages but currently it is only working for the english pages. I've also tried setting up some exact redirects for translated pages but they don't work (even when the equivalent for english does work). 

Not sure why redirects seem to be limited to English only and can't find any helpful documentation. Not sure what the best course of action is here
